lms/djangoapps/onboarding_survey/forms.py

```diff
@@ -395,6 +403,34 @@ def save(self, user=None, commit=True):
                 prev_org.save()
 
         extended_profile.organization = organization_to_assign
+
+        is_extended_profile = None
+        try:
+            is_extended_profile = user.extended_profile
+        except:
+            pass
+
+        if not is_extended_profile and extended_profile.org_admin_email:
+            admin_user = None
+
+            try:
+                admin_user = User.objects.get(email=extended_profile.org_admin_email)
+            except:
+                pass
+
+            if admin_user:
```

Comment:
move all this code in try block and catch related exceptions if you think they can occur